src/database/migrations/20201007150915-create-users.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Users', {
+      id: { allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
```

Comment:
change the id to uuid not an integer(autoIncrement)

Reply:
changed to UUID

Reply:
Done